privateSBOMExchange/src/petra/lib/models/__init__.py

```diff
@@ -348,41 +346,57 @@ def visit_sbom_node(self, node:SbomNode):
 class PrintVisitor:
     """Visitor that prints the data and hash of each node."""
     def visit_field_node(self, node:FieldNode):
-        print(f"{node.encrypted_data};{node.field_name}:{node.field_value};{node.policy}")
+        log(f"{node.encrypted_data};{node.field_name}:{node.field_value};{node.policy}")
 
         try:
-            print(f"Hash: {node.hash.hex()}")
+            log(f"Hash: {node.hash.hex()}")
         except AttributeError:
             pass
-            #print("Hashes have not been calculated, you need to first visit the tree using the Merkle Visitor")
+            #log("Hashes have not been calculated, you need to first visit the tree using the Merkle Visitor")
 
     def visit_complex_node(self, node:ComplexNode):
-        print(f"{node.encrypted_data};{node.complex_type};{node.policy}") 
+        log(f"{node.encrypted_data};{node.complex_type};{node.policy}") 
         try:
-            print(f"Hash: {node.hash.hex()}")
+            log(f"Hash: {node.hash.hex()}")
         except AttributeError:
             pass
-            #print("Hashes have not been calculated, you need to first visit the tree using the Merkle Visitor")
+            #log("Hashes have not been calculated, you need to first visit the tree using the Merkle Visitor")
 
         for child in node.children:
             child.accept(self)
             
     def visit_sbom_node(self, node:SbomNode):
-        print(f"SBOM: {node.purl}") 
+        log(f"SBOM: {node.purl}") 
         try:
-            print(f"Hash: {node.hash.hex()}")
+            log(f"Hash: {node.hash.hex()}")
         except AttributeError:
             pass
-            #print("Hashes have not been calculated, you need to first visit the tree using the Merkle Visitor")
+            #log("Hashes have not been calculated, you need to first visit the tree using the Merkle Visitor")
+            
+        for child in node.children:
+            child.accept(self)        
+
+
+class PrintFieldNode:
+    """Visitor that prints the data and hash of each node."""
```

Comment:
This class only seems to  exist for debugging/testing. Can we please move this out of the `modules` subpackage? The class name and docstring also do not accurately reflect what the class does, please update these to avoid confusion with the `PrintVisitor`.